[2025-03-08T05:47:24.847+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2025-03-07T00:00:00+00:00 [queued]>
[2025-03-08T05:47:24.855+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2025-03-07T00:00:00+00:00 [queued]>
[2025-03-08T05:47:24.856+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-03-08T05:47:24.867+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2025-03-07 00:00:00+00:00
[2025-03-08T05:47:24.871+0000] {standard_task_runner.py:57} INFO - Started process 7245 to run task
[2025-03-08T05:47:24.874+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2025-03-07T00:00:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/postgres_operator.py', '--cfg-path', '/tmp/tmprvc81nye']
[2025-03-08T05:47:24.877+0000] {standard_task_runner.py:85} INFO - Job 42: Subtask insert_into_table
[2025-03-08T05:47:24.924+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2025-03-07T00:00:00+00:00 [running]> on host afc6f8faa09c
[2025-03-08T05:47:25.008+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-07T00:00:00+00:00'
[2025-03-08T05:47:25.009+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2025-03-07', 'dag_with_postgres_operator_v03')
        
[2025-03-08T05:47:25.019+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-08T05:47:25.028+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-08T05:47:25.033+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2025-03-07', 'dag_with_postgres_operator_v03')
        , parameters: None
[2025-03-08T05:47:25.035+0000] {sql.py:427} INFO - Rows affected: 1
[2025-03-08T05:47:25.045+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20250307T000000, start_date=20250308T054724, end_date=20250308T054725
[2025-03-08T05:47:25.086+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-08T05:47:25.101+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
